### PR TITLE
[devtools-translate] Spanish - What's New in DevTools (Chrome 101)

### LINKED ISSUE #2578: [devtools-translate] Spanish - What's New in DevTools (Chrome 101)
Post: https://developer.chrome.com/blog/new-in-devtools-101/
Translated by: Apr 20

Translator: @midudev 
Reviewers: @Caballerog @and-oli

### PR DESCRIPTION
**Pull request by midudev**
Fixes #2578

Changes proposed in this pull request:

- Add Spanish translation for Chrome 101 changes articles.

### CODE REVIEW

**Comment by midudev** [comment]
Nothing I could do regarding linter problems as they're from the original content.

Sorry for the mess with the commit history. 😅

**Comment by jecfish** [comment]
Thanks @midudev! Over to @Caballerog for review.

File: site/es/blog/new-in-devtools-101/index.md [hunk 1]
```diff
@@ -15,62 +15,79 @@ tags:
 draft: true
```
Comment (jecfish) [inline site/es/blog/new-in-devtools-101/index.md:15]: remove draft
```suggestion

```

**Review by jecfish — CHANGES_REQUESTED** [review]

File: site/es/blog/new-in-devtools-101/index.md [hunk 2]
```diff
@@ -5,72 +5,88 @@ authors:
...
   - new-in-devtools
   - devtools
   - chrome-101
-draft: true
 ---
 
-_Gracias [Carlos Caballero](https://carloscaballero.io) por la traducción y [Miguel Ángel](https://midu.dev) por la revisión._
+_Gracias [Miguel Ángel](https://midu.dev) por la traducción y [Carlos Caballero](https://carloscaballero.io) por la revisión._
 
 {% include 'partials/devtools/es/banner.md' %}
```
Comment (jecfish) [inline site/es/blog/new-in-devtools-101/index.md:20]:
```suggestion

{% YouTube id='u9GRAliBrM8' %}
```

File: site/es/blog/new-in-devtools-101/index.md [hunk 3]
```diff
@@ -5,72 +5,89 @@ authors:
...
 
 {% Video src="video/dPDCek3EhZgLQPGtEG3y0fTn4v82/Jy7NEDZs6XJb90EWqETj.mp4", class="screenshot", autoplay=true, controls=true, loop=true, muted=true %}
 
 <!-- Apart from that, you can export the recording as well. After [recording a user flow](/docs/devtools/recorder/#record), click on the export button. There are 3 export options: -->
+Además de esto, puede exportar también la grabación. Después de [grabar un flujo de usuario](/docs/devtools/recorder/#record), haga clic en el botón exportar. Hay 3 opciones de exportación:
 
 <!-- - **Export as a JSON file**. Download the recording as a JSON file. -->
+- **Exportar como un archivo JSON**. Descargue la grabación como un archivo JSON.
 <!-- - **Export as a @puppeteer/replay script**. Download the recording as a [Puppeteer Replay](https://github.com/puppeteer/replay) script.  -->
+- **Exportar como un script de @puppeteer/replay**. Descargue la grabación como un script de [Puppeteer Replay](
```
Comment (Caballerog) [inline site/es/blog/new-in-devtools-101/index.md:39]: The link is missing: 
[Puppeteer Replay](https://github.com/puppeteer/replay)

File: site/es/blog/new-in-devtools-101/index.md [hunk 4]
```diff
@@ -80,29 +97,41 @@ Chromium issues: [1296855](https://crbug.com/1296855), [https://crbug.com/130340
...
+Por ejemplo, puede usar los informes de **tiempo de espera** para analizar las interacciones de usuario. Abra esta página [demo](https://coffee-cart.netlify.app/). Seleccione el modo de **Tiempo de Espera** y haga clic en **Iniciar tiempo de bloqueo**. En la página, haga clic en un café y termine el tiempo de bloqueo. Lea el informe para encontrar el [Tiempo de bloqueo total](https://web.dev/tbt/) y [Desplazamiento acumulado del diseño](https://web.dev/cls/) que fueron causados por la interacción.
 
 <!-- Each mode has its own unique use cases, benefits, and limitations. Please refer to the [Lighthouse documentation](https://github.com/GoogleChrome/lighthouse/blob/master/docs/user-flows.md) for more information. -->
+Cada mode tiene sus propios casos de uso, beneficios y limitaciones. Por favor, consulte la [documentación de Lighthouse](https://github.com/GoogleChrome/lighthouse/blob/master/docs/user-flows.md) para más información.
```
Comment (Caballerog) [inline site/es/blog/new-in-devtools-101/index.md:132]: mode => "modo"

**Review by Caballerog — COMMENTED** [review]
I have only found two small errors. The rest of the translation is really good!

File: site/es/blog/new-in-devtools-101/index.md (hunk 3 above)
Comment (Caballerog) [reply site/es/blog/new-in-devtools-101/index.md]: Perfect!

File: site/es/blog/new-in-devtools-101/index.md (hunk 4 above)
Comment (Caballerog) [reply site/es/blog/new-in-devtools-101/index.md]: Perfect!

**Comment by Caballerog** [comment]
LGTM!

**Review by jecfish — APPROVED** [review]

**Comment by jecfish** [comment]
Gracias both!

**Pull request merged**